src/main/java/com/segment/analytics/android/integrations/flurry/FlurryIntegration.java

```diff
@@ -123,7 +123,6 @@ public void onActivityStopped(Activity activity) {
   public void screen(ScreenPayload screen) {
     super.screen(screen);
     // todo: verify behaviour here, iOS SDK only does pageView, not event
-    FlurryAgent.onPageView();
     logger.verbose("FlurryAgent.onPageView();");
```

Comment:
If we remove `FlurryAgent.onPageView()`, then there's no need to keep the log just after.